src/mappings/roundsManager.ts

```diff
@@ -177,6 +179,59 @@ export function newRound(event: NewRound): void {
     }
   }
 
+  // Update rolling fee windows for tracked broadcasters (gateways)
+  let trackedBroadcasters = protocol.activeBroadcasters;
+  let activeBroadcasters: string[] = [];
+  if (trackedBroadcasters && trackedBroadcasters.length) {
+    for (let i = 0; i < trackedBroadcasters.length; i++) {
+      let broadcaster = Broadcaster.load(trackedBroadcasters[i]);
+
+      if (broadcaster) {
+        // --- Get the 30, 60, 90 day sums of volume ---
+        let broadcasterThirtyDaySum = ZERO_BD;
+        let broadcasterSixtyDaySum = ZERO_BD;
+        let broadcasterNinetyDaySum = ZERO_BD;
+
+        // capped at <90 - broadcaster days are ordered newest first
+        let broadcasterDays = broadcaster.broadcasterDays;
+        let broadcasterDaysLength =
+          broadcasterDays.length > 90 ? 90 : broadcasterDays.length;
+        for (let j = 0; j < broadcasterDaysLength; j++) {
```

Comment:
Functionally this works, but I'm always apprehensive to include nested loops within subgraphs.

I think for now if the subgraph syncs in a timely manner this is okay. We can look at alternatives that remove the necessity for these nested loops in the audit, as these may require broader changes beyond this block of code.